jet_tracking/devices.py

```diff
@@ -866,3 +910,24 @@ class Diffract(Device, _TableMixin):
                         doc='Diffraction periodogram Frequency analysis amplitude array')
     state = Cpt(EpicsSignal, ':STATE',
                 doc='State of diffraction analysis')
+
+
+# classes used for jet tracking testing
+class JTInput(Device):
+    nframe = Cpt(EpicsSignal, ':NFRAME', doc='number of frames passed')
+    i0 = Cpt(EpicsSignal, ':i0', doc='Wave8')
+    evtcode = Cpt(EpicsSignal, ':EVTCODE', doc='event code')
+    mtr = Cpt(EpicsSignal, ':MTR', doc='motor position')
+    mtr_prec = Cpt(EpicsSignal, ':MTR_PREC', doc='motor precision')
+
+
+class JTOutput(Device):
+    nframe = Cpt(EpicsSignal, ':NFRAME', doc='number of frames used')
+    det = Cpt(EpicsSignal, ':DET', doc='detector intensity')
+    i0 = Cpt(EpicsSignal, ':I0', doc='Wave8')
```

Comment:
```suggestion
    i0 = Cpt(EpicsSignal, ':I0', doc='Wave8 intensity')
```
Furthermore, can we more specific about what these intensities are in the docstring?  I suspect this is a summation of all the diodes.  In which case it could say "Wave8 intensity sum" or similar.  To get really particular, we should also indicate that this is a physical quantity expressed in arbitrary units, since there is no pulse energy calibration.  Not sure that ECS is rigorous about this though...

Reply:
I'm not even sure we should really call this 'wave8' as in theory we could decide to slap a PIPS diode someplace to get a better measurement of the i0, e.g. in XPP. Longer comment about naming somewhere else....